Maeliosa/dappnode v2 (#382)
ObolNetwork/obol-docs: docs/start/quickstart_group.md

          <img src="/img/StartDKG.png" alt="Start DKG" />
          <ol>
            <li>
              After all operators has signed with their wallet and has provided an ENR from the DappNode info tab, the launchpad page will guide operators to the DKG ceremony. Click continue & navigate to 'Dappnode/Avado' tab where the cluster definition file URL is presented. 
              <img src="/img/DappnodeTab.png" alt="Dappnode Tab" />
            </li>
             <li>
              To run the Distributed Key Generation ceremony using a DappNode, paste the URL into the Obol Package interface. Go to the 'Config' tab > select 'URL' from the dropdown menu > paste the Definition File URL, you retrieved from the launchpad, into the validator cluster-Num field which should match the ENR cluster-x. Example: If you picked ENR1 for signing, then you should paste the URL in Cluster-1. Finally, click the update button at the bottom of the page. 
              <img src="/img/CopyURL.png" alt="Copy URL" />
              <img src="/img/PasteURL.png" alt="Paste URL" />
              <img src="/img/SelectURL.png" alt="Select URL from dorpdown" />
            </li>
              <li>

>               After DappNode updates the definition file, go back into the info tab & restart the Charon validator container. 

Reviewer: ```suggestion
              After DappNode records the cluster definition URL, go back to the 'Info' tab and restart the Charon container. 
```

              <img src="/img/RestartContainer.png" alt="Restart Charon validator container" />
            </li>
            <li>

>               The node is now ready and will attempt to complete the DKG. You can monitor the DKG progress via the logs tab. Once all clients in the cluster can establish a connection with one another and they each complete a handshake (confirm everyone has a matching `cluster_definition_hash`), the ceremony begins.

Reviewer: ```suggestion
              The node is now ready and will attempt to complete the DKG. You can monitor the DKG progress via the 'Logs' tab of the package. Once all clients in the cluster can establish a connection with one another and they each complete a handshake (confirm everyone has a matching <code>cluster_definition_hash</code>), the key generation ceremony begins.
```

              <img src="/img/ConnectPeers.png" alt="Connect to peers in logs tab" />
            </li>
             <li>
              Example of DKG ceremony competed log. 
              <img src="/img/DKGExample.png" alt="Connect to peers in logs tab" />
            </li>
            </ol>
  
  </TabItem>
</Tabs>

> :::warning

Reviewer: This warning is a bit different for the dappnode instructions, maybe we pull this warning up into the tabs to have different wording for dappnode? Probably wouldn't render correctly though so it might be more trouble than it is worth.
